testing/test-scripts/bookinfo-default-route-productpage.yaml

```diff
@@ -6,3 +6,11 @@ rules:
     backends:
     - tags:
       - version=v1
+      resilience:
```

Comment:
Even though these aren't things that are directly triggered, these files do get used by the demos, so why don't we have less strange values? Otherwise people will look at them and be confused about the significance of 1111 vs simply 1000.

1111 -> 1000
222 -> 200
1234 -> 1500
etc